components/resume/job/job.tsx

```diff
@@ -1,12 +1,17 @@
 import React, { FC } from 'react';
-import { Job } from '../types';
-import Position from './position';
+import type { Job } from '../types';
 import DateRange from './dateRange/dateRange';
+import Position from './position';
 
 const Job: FC<Job> = ({ companyName, positions, highlights, startDate, endDate }) => {
     return <div>
-        <h3>{companyName}</h3>
-        <div><DateRange start={startDate} end={endDate} /></div>
+        <div className="resume__job">
+            <div className="resume__job__company">
+                <div className="resume__job__company__info">{companyName}</div>
```

Comment:
This is where BEM get's kinda funky and makes me consider going with css in js. I'm probably going overboard with the sub module names here as well. Also, with BEM coming up with semantic names for layout related styles takes time and not sure if it's useful or even clear.

Not a huge deal yet, but may pivot to css in js in the future to simplify all this a bit, but holding off for now till I see how print styles would work out. Not sure if css in js (pick a framework) handle that well.